ui/src/stores/cookieStore.ts

```diff
@@ -0,0 +1,41 @@
+import { defineStore } from "pinia";
+import { CookieState } from "@/stores/types/cookieState";
+
+import { isObjectHasKeys } from "@im-library/helpers/DataTypeCheckers";
+
+export const useCookieStore = defineStore("cookie", {
+    state: (): CookieState => ({
+        cookiesEssentialAccepted: localStorage.getItem("cookiesEssentialAccepted") === "true" ? true : false,
```

Comment:
these are really user related items, and would end up in the user database. Move to userstore